packages/email-templates/package.json

```diff
@@ -8,7 +8,11 @@
   ],
   "main": "dist",
   "devDependencies": {
-    "typescript": "4.9.4"
+    "@babel/plugin-transform-runtime": "7.19.6",
+    "@vitest/coverage-c8": "0.28.3",
+    "regenerator-runtime": "0.13.11",
```

Comment:
Are these new dependencies?

Reply:
we dont need them actually, just `vitest` here